LICENSE.txt

```diff
@@ -1,8 +1,9 @@
-Copyright INRIA and ENPC (contributors: Geoffray ADDE, Maureen CLERC,
-Alexandre GRAMFORT, Renaud KERIVEN, Jan KYBIC, Perrine LANDREAU,
-Emmanuel OLIVI, Theodore PAPADOPOULO, Maureen.Clerc.AT.inria.fr, keriven.AT.certis.enpc.fr, kybic.AT.fel.cvut.cz, papadop.AT.inria.fr, Alexandre.Gramfort.AT.inria.fr, Emmanuel.Olivi.AT.inria.fr)
+© Copyright INRIA and ENPC
+Contributors: Geoffray ADDE, Maureen CLERC, Alexandre GRAMFORT,
+              Renaud KERIVEN, Jan KYBIC, Perrine LANDREAU,
+              Emmanuel OLIVI, Theodore PAPADOPOULO.

```

Comment:
I'd rather not have to keep updating this, can we just say `OpenMEEG developers` or something?

Otherwise, please add my name at least. But to be complete, something like this would get you a list:
```
$ git shortlog -s --group=author --group=trailer:co-authored-by
   388  Alexandre Gramfort
     1  Boris Burle
     2  Dang Kai
     3  Daniel Miklody
   566  Emmanuel Olivi
   114  Eric Larson
     1  J-H-Cho
    48  Joan Massich
     2  Kyle Beauchamp
     1  Marmaduke Woodman
    76  Maureen Clerc
    12  Perrine Landreau
     5  Renaud Keriven
     1  Sarthak Pati
     1  Sylvain Vallaghe
     1  The Gitter Badger
  1020  Theodore Papadopoulo
     2  Thierry Guillemot
     1  Thomas Baag
     4  Yaroslav Halchenko
     1  j-l-s
     1  lazyparser
```
And there are some names in there we'd need to ping people about...